data/meson.build

```diff
@@ -4,6 +4,8 @@ install_data('org.erikreider.swaync.gschema.xml',
 
 subdir('icons')
 
+gnome.compile_schemas()
```

Comment:
Doesn't seem to update unless I remove the data dir in the compilation dir. Only works the first time I change the schema

Otherwise LGTM :)

Reply:
That is true. I don't really know how to solve that, but I still think this is useful for development on a system without swaync installed (or on nix, where the schemas are isolated).

Reply:
Might require something like this: https://mesonbuild.com/Reference-manual_functions.html#custom_target